src/screens/FeedScreen/Content.js

```diff
@@ -6,6 +6,7 @@ import {Dimensions, Platform, Pressable, StyleSheet, Text, View} from 'react-nat
 // eslint-disable-next-line import/no-extraneous-dependencies
 import {useNavigation, useRoute} from '@react-navigation/native';
 
+import LinearGradient from 'react-native-linear-gradient';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `LinearGradient` from 'react-native-linear-gradient' is new in this hunk. Ensure that the package 'react-native-linear-gradient' is installed and properly configured in your project. If not, it may cause a runtime error.
```diff
+ import LinearGradient from 'react-native-linear-gradient';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->